### PR TITLE
fix set current page

### PR DESCRIPTION
**Pull request by paodb**

### CODE REVIEW

File: src/vcf-pdf-viewer.js [hunk 1]
```diff
@@ -628,10 +627,13 @@ class PdfViewerElement extends
     }
 
     setCurrentPage(value) {
-        if (!this.__viewer || this.__loading) {
-            this.$.currentPage.value = value;
-        } else {
-            this.__pageChange();
+        if (value != undefined) {
+            if (!this.__viewer || this.__loading) {
```
Comment (mlopezFC) [inline src/vcf-pdf-viewer.js:631]: Wouldn't be better to take this outside the if clause, given that it is repeated?:
this.$.currentPage.value = value;

**Review by mlopezFC — CHANGES_REQUESTED** [review]
If I understood correctly, the if clause can be improved

File: src/vcf-pdf-viewer.js (hunk 1 above)
Comment (paodb) [reply src/vcf-pdf-viewer.js:631]: Let me take a look

Comment (paodb) [reply src/vcf-pdf-viewer.js]: I did some changes in the method as it was not working as expected

File: src/vcf-pdf-viewer.js [hunk 2]
```diff
@@ -628,11 +628,10 @@ class PdfViewerElement extends
     }
 
     setCurrentPage(value) {
-        if (!this.__viewer || this.__loading) {
+        if (value != undefined) {
             this.$.currentPage.value = value;
-        } else {
-            this.__pageChange();
         }
+        this.__pageChange();
```
Comment (mlopezFC) [inline src/vcf-pdf-viewer.js:634]: Just a question: should `__pageChange()` be called given that `this.$.currentPage.value` might have no changes?

**Review by mlopezFC — COMMENTED** [review]
Just one question

File: src/vcf-pdf-viewer.js (hunk 2 above)
Comment (paodb) [reply src/vcf-pdf-viewer.js:634]: The value is set in the previous call or by entering in it in the  page numbers textfield, or by using the arrows or by selecting a thumbnail. By default is the first page.

Comment (mlopezFC) [reply src/vcf-pdf-viewer.js:634]: 👍

**Review by mlopezFC — APPROVED** [review]
LGTM

**Pull request merged**